src/main/java/br/com/openenade/api/universidade/UniversidadeController.java

```diff
@@ -44,6 +46,15 @@ public void postUniversidade(@Valid @RequestBody Universidade universidade) {
         return this.service.getUniversidadeByCodigoIES(codigoIES);
     }
     
+    @ResponseBody
+    @GetMapping(path = "/{codigoIES}/{campus}")
+    public Universidade getUniversidadeById(@PathVariable(name = "codigoIES") Long codigoIES,
+    		@PathVariable(name = "campus") Municipio campus){
```

Comment:
Why the "campus" path variable is a Municipio? I think that it should be the "código municipio".